src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1886,6 +1886,44 @@ namespace UnitarySystems {
 
             SizingMethod = DataHVACGlobals::SystemAirflowSizing;
 
+            MSHPIndex = this->m_DesignSpecMSHPIndex;
+            if (MSHPIndex > -1) {
+                // set no load air flow ratio local var
+                Real64 NoLoadCoolingAirFlowRateRatio = 1.0;
+                Real64 NoLoadHeatingAirFlowRateRatio = 1.0;
```

Comment:
Move these up outside the if (MSHPIndex > -1).

Reply:
Done